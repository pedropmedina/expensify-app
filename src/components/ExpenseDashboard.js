import React from 'react';
import { connect } from 'react-redux';

import ExpenseList from './ExpenseList';
import ExpenseListFilters from './ExpenseListFilters';

class ExpenseDashboard extends React.Component {
	render() {
		return (
			<React.Fragment>
				<ExpenseListFilters />
				<ExpenseList />
			</React.Fragment>
		);
	}
}

export default ExpenseDashboard;
