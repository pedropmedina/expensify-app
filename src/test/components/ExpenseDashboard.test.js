import React from 'react';
import { shallow } from 'enzyme';

import ExpenseDashboard from '../../components/ExpenseDashboard';

test('should render ExpenseDashboard component correctly', () => {
	const wrapper = shallow(<ExpenseDashboard />);
	expect(wrapper).toMatchSnapshot();
});
