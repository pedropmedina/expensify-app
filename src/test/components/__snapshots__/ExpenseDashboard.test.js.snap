// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseDashboard component correctly 1`] = `
<React.Fragment>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</React.Fragment>
`;
